src/main/java/de/zalando/spring/cloud/config/aws/kms/KmsProperties.java

```diff
@@ -23,6 +23,46 @@
      */
     private String region;
 
+    /**
+     * <strong>Optional</strong> service endpoint and signing region of AWS KMS that you would like to route to.
+     * If provided, must supply either a custom created VPC Endpoint or one of the KMS Endpoints listed <a href="https://docs.aws.amazon.com/general/latest/gr/rande.html#kms_region">here</a>.
+     * In the event that both region and endpoint properties are both supplied, region will be ignored as region is derived from the service endpoint.
+     */
+    private Endpoint endpoint;
+    
+    @ConfigurationProperties("endpoint")
```

Comment:
This annotation can be omitted. See docs about [nested properties](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#configuration-metadata-nested-properties).